app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```diff
@@ -0,0 +1,39 @@
+.gem-c-big-number {
+  margin-bottom: govuk-spacing(3);
+}
+
+.gem-c-big-number__value {
+  @include govuk-font($size: 80, $weight: bold);
+}
+
+.gem-c-big-number__label {
+  @include govuk-font($size: 16, $weight: bold, $line-height: 2);
+
+  &:before {
+    content: "";
+    display: block;
```

Comment:
The reason for this pseudo element is documented nicely in the YML file but I wonder if it would be worth putting a short comment in here as well? The reason being that if a developer might not think to consult the YML file before removing this bit (if they thought it wasn't useful).